Dockerfile.rhtap

```diff
@@ -0,0 +1,20 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.19 as builder
```

Comment:
You'll want to edit this to point to brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 or whichever go version is in your midstream file.

Reply:
Updated.